### PR TITLE
PEP-8 bodhi.server.scripts and enforce style.

### PR DESCRIPTION
**Pull request by bowlofeggs**

### CODE REVIEW

File: bodhi/server/scripts/expire_overrides.py [hunk 1]
```diff
@@ -56,7 +56,7 @@ def main(argv=sys.argv):
         now = datetime.utcnow()
 
         overrides = db.query(BuildrootOverride)
-        overrides = overrides.filter(BuildrootOverride.expired_date == None)
+        overrides = overrides.filter(is_(BuildrootOverride.expired_date, None))
```
Comment (bowlofeggs) [inline bodhi/server/scripts/expire_overrides.py]: I think this should actually be `BuildrootOverride.expired_date.is_(None)` instead.

Comment (pypingou) [reply bodhi/server/scripts/expire_overrides.py]: that's how you used it elsewhere indeed

**Comment by pypingou** [comment]
Except for the line you pointed out, looks good to me

File: bodhi/server/scripts/expire_overrides.py (hunk 1 above)
Comment (bowlofeggs) [reply bodhi/server/scripts/expire_overrides.py]: This is fixed.

**Pull request merged**